migrations/20220830190801-add-flag-to-system-activities.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(`
+      UPDATE "Activities" SET data = jsonb_set("data", '{isSystem}', 'true') WHERE "Activities".type IN ('taxform.request', 'collective.expense.error', 'collective.virtualcard.missing.receipts', 'collective.monthlyreport', 'payment.failed', 'payment.creditcard.expiring', 'virtualcard.charge.declined');
```

Comment:
It's better to indent your queries, makes them easier to read & review

```suggestion
      UPDATE "Activities"
      SET data = jsonb_set("data", '{isSystem}', 'true')
      WHERE "Activities".type IN (
        'taxform.request',
        'collective.expense.error',
        'collective.virtualcard.missing.receipts',
        'collective.monthlyreport',
        'payment.failed',
        'payment.creditcard.expiring',
        'virtualcard.charge.declined'
      );
```